src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/MicrosoftTeamsTransport.php

```diff
@@ -56,11 +57,21 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
         }
 
+        if ($message->getOptions() && !$message->getOptions() instanceof MicrosoftTeamsOptions) {
+            throw new LogicException(sprintf('The "%s" transport only supports instances of "%s" for options.', __CLASS__, MicrosoftTeamsOptions::class));
+        }
+
+        $options = ($opts = $message->getOptions()) ? $opts->toArray() : [];
```

Comment:
We check if options are null twice here.

Reply:
It looks like a copy paste thingy:
![CleanShot 2021-04-12 at 20 47 46](https://user-images.githubusercontent.com/995707/114445447-5ff00380-9bd0-11eb-92ec-9fa0f5e3679f.png)

Lets keep it for now in this PR and fix all of them together?